pkg/apply/info/info_helper.go

```diff
@@ -15,12 +15,12 @@ import (
 
 // InfoHelper provides functions for interacting with Info objects.
 type InfoHelper interface {
-	// UpdateInfos sets the mapping and client for the provided Info
+	// UpdateInfo sets the mapping and client for the provided Info
 	// objects. This must be called at a time when all needed resource
 	// types are available in the RESTMapper.
-	UpdateInfos(infos []*resource.Info) error
+	UpdateInfo(infos *resource.Info) error
```

Comment:
We should change the parameter here to singular.

Reply:
fixed